### ISSUE TITLE
Fails when no documentation

### ISSUE THREAD

**Issue by OEvgeny** [fortunejs/fortune-http#10]
While trying to set up `fortune-http` I found that it needs the `documentation` field to be an `object` to start to work correctly.

```
TypeError: Cannot use 'in' operator to search for 'name' in undefined
    at E:\work\node_modules\fortune-http\lib\html_serializer.js:593:15
    at HtmlSerializer.renderCollection (E:\work\node_modules\fortune-http\lib\html_serializer.js:647:4)
    at HtmlSerializer.processResponse (E:\work\node_modules\fortune-http\lib\html_serializer.js:239:10)
    at E:\work\node_modules\fortune-http\lib\index.js:357:12
    at <anonymous>
```

**Comment by OEvgeny** [comment]
My `package.json` deps

```
    "fortune": "^5.2.11",
    "fortune-http": "^1.2.7",
    "fortune-nedb": "^1.1.5",
```

**Comment by gr0uch** [comment]
Oops, it's fixed in https://github.com/fortunejs/fortune-http/commit/716ed1b3ac79c3d973af21490b19bdf60f6c6183. Published as `1.2.8`.

**Comment by OEvgeny** [comment]
@daliwali wow thanks!
